build-system/tasks/performance/README.md

```diff
@@ -5,3 +5,5 @@ Use `gulp performance` to run this test. It measures performance for the current
 The `config.json` file contains settings. If you set `headless` to `false`, you will be able to watch the test run in Chrome. The default value for `runs` is 50, which takes ~10 minutes to run, and generates results with ~5% margin of error. Differences larger than this should be investigated. The `defaultHandler` within the `handlers` list measures the basic pageview metrics (visible, first paint, etc...). The other handlers within the list can track different metrics (analytics request time, amount of analytics requests failed, etc...). Each handler within the handlers list must have a `handlerName` as well as a `urls` array (can be empty).
 
 This test measures the following metrics: `visible`, `first paint`, `first contentful paint`, `largest contentful paint`, `time to interactive`, `max first input delay`, `cumulative layout shift`, `analytics request delay`, and `failed percentage of analytics requests`.
+
+To verify that the tests and test pages are loading properly, use `headless: false` in the config and add `devtools: true` to the options in `puppeteer.launch()`.
```

Comment:
Hmm, would it be good to add this as a `--dev` or `--debug` flag to the task so that devs don't need to modify the config file?

Reply:
https://github.com/ampproject/amphtml/pull/28689

Reply:
Yes! That would be awesome :D